Terminal.Gui/ConsoleDrivers/Net/NetDriver.cs

```diff
@@ -0,0 +1,902 @@
+//
+// NetDriver.cs: The System.Console-based .NET driver, works on Windows and Unix, but is not particularly efficient.
+//
+
+using static Terminal.Gui.ConsoleDrivers.ConsoleKeyMapping;
+using static Terminal.Gui.ConsoleDrivers.Net.NetEvents;
+
+namespace Terminal.Gui.ConsoleDrivers.Net;
+
+using System.Diagnostics.CodeAnalysis;
+using System.Runtime.InteropServices;
+using static EscSeqUtils;
+
+internal sealed class NetDriver : ConsoleDriver
+{
+    private const int COLOR_BLACK = 30;
+    private const int COLOR_BLUE = 34;
+    private const int COLOR_BRIGHT_BLACK = 90;
+    private const int COLOR_BRIGHT_BLUE = 94;
+    private const int COLOR_BRIGHT_CYAN = 96;
+    private const int COLOR_BRIGHT_GREEN = 92;
+    private const int COLOR_BRIGHT_MAGENTA = 95;
+    private const int COLOR_BRIGHT_RED = 91;
+    private const int COLOR_BRIGHT_WHITE = 97;
+    private const int COLOR_BRIGHT_YELLOW = 93;
+    private const int COLOR_CYAN = 36;
+    private const int COLOR_GREEN = 32;
+    private const int COLOR_MAGENTA = 35;
+    private const int COLOR_RED = 31;
+    private const int COLOR_WHITE = 37;
+    private const int COLOR_YELLOW = 33;
+    private NetMainLoop _mainLoopDriver = null!;
+    public bool IsWinPlatform { get; private set; }
+    public NetWinVTConsole? NetWinConsole { get; private set; }
+
+    public override void Refresh ()
+    {
+        UpdateScreen ();
+        UpdateCursor ();
+    }
+
+    public override void SendKeys (char keyChar, ConsoleKey key, bool shift, bool alt, bool control)
```

Comment:
This API should really be deprecated. 

It was poorly designed and is weird. Tests should use `Application.OnKeyDown()` to simulate input.

Reply:
I support that wholeheartedly.

There's likely to be a fair amount of old stuff ripped out as I move along in these.

As in the other comment, though, how about I address that when I branch for that driver specifically, or if it naturally requires a heavy hand due to coupling or something?